Lib/symtable.py

```diff
@@ -258,7 +261,7 @@ def is_declared_global(self):
     def is_local(self):
         """Return *True* if the symbol is local.
         """
-        return bool(self.__scope in (LOCAL, CELL))
+        return bool(self.__scope in (LOCAL, CELL) or self.__module_scope and self.__flags & DEF_BOUND)
```

Comment:
Ditto. (It seems that there are other PEP 8 issues in this PR as well.)

Reply:
Will do a PEP 8 pass before merging. Thanks for the review!